application/src/main/java/org/togetherjava/tjbot/Application.java

```diff
@@ -81,6 +82,10 @@ public static void runBot(String token, Path databasePath) {
             jda.awaitReady();
             logger.info("Bot is ready");
 
+            // TODO This should be moved into some proper command system instead (see GH issue #235
+            // which adds support for routines)
```

Comment:
Is the todo needed?

Reply:
yes. otherwise people might forget about this or dont understand why this code is there in the way it is there right now (which is not really the best way to write that code).